src/core/ucc_lib.c

```diff
@@ -27,16 +27,6 @@ static ucc_config_field_t ucc_lib_config_table[] = {
 UCC_CONFIG_REGISTER_TABLE(ucc_lib_config_table, "UCC", NULL, ucc_lib_config_t,
                           &ucc_config_global_list)
 
-static inline ucc_status_t ucc_cl_component_is_loaded(ucc_cl_type_t cl_type)
```

Comment:
Is this related changes as well?

Reply:
yes, clang says that function is unused and prints error during compilation